<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWidget</class>
 <widget class="QWidget" name="SettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>437</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Einstellungen</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="mainVerticalLayout">
     <item>
      <widget class="QTabWidget" name="settingsTabWidget">
       <property name="currentIndex">
        <number>3</number>
       </property>
       <widget class="QWidget" name="generalTab">
        <attribute name="title">
         <string>Allgemein</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="generalVerticalLayout">
           <property name="sizeConstraint">
            <enum>QLayout::SetDefaultConstraint</enum>
           </property>
           <item alignment="Qt::AlignTop">
            <widget class="QGroupBox" name="programOptionsGroupBox">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>100</height>
              </size>
             </property>
             <property name="title">
              <string>Programmoptionen</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                 <widget class="QCheckBox" name="startWithWindowsCheckBox">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string>Mit Windows starten</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="keepInForegroundCheckBox">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string>Im Vordergrund halten</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="designTab">
        <attribute name="title">
         <string>Design</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_13">
         <item>
          <layout class="QVBoxLayout" name="designVerticalLayout">
           <item alignment="Qt::AlignTop">
            <widget class="QGroupBox" name="designGeneralGroupBox">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>100</height>
              </size>
             </property>
             <property name="title">
              <string>Allgemein</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_19">
              <item>
               <layout class="QVBoxLayout" name="designGeneralVerticalLayout">
                <item>
                 <widget class="QCheckBox" name="showSecondsCheckBox">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string>Sekunden anzeigen</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item alignment="Qt::AlignTop">
            <widget class="QGroupBox" name="designColorGroupBox">
             <property name="title">
              <string>Farbauswahl</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_21">
              <item>
               <layout class="QVBoxLayout" name="designColorVerticalLayout">
                <item>
                 <widget class="QPushButton" name="foregroundColorButton">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string>Vordergrundfarbe auswählen</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="backgroundColorButton">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string>Hintergrundfarbe auswählen</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="foregroundColorLabel">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string>Vordergrundfarbe</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="backgroundColorLabel">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string>Hintergrundfarbe</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="designVerticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="timeAnnouncementTab">
        <attribute name="title">
         <string>Zeitansage</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QVBoxLayout" name="timeAnnouncementVerticalLayout">
           <item alignment="Qt::AlignTop">
            <widget class="QGroupBox" name="timeAnnouncementGeneralGroupBox">
             <property name="title">
              <string>Allgemein</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_8">
              <item>
               <layout class="QVBoxLayout" name="timeAnnouncementGeneralVerticalLayout">
                <item>
                 <widget class="QCheckBox" name="timeAnnouncementCheckBox">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string>Zeitansage aktivieren</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item alignment="Qt::AlignTop">
            <widget class="QGroupBox" name="timeAnnouncementIntervalGroupBox">
             <property name="title">
              <string>Häufigkeit</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_9">
              <item>
               <layout class="QVBoxLayout" name="timeAnnouncementIntervalVerticalLayout">
                <item>
                 <widget class="QRadioButton" name="everyQuarterRadioButton">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string>Jede Viertelstunde</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="everyHalfHourRadioButton">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string>Jede halbe Stunde</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="everyHourRadioButton">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string>Jede Stunde</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="timeAnnouncementVerticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="weatherTab">
        <attribute name="title">
         <string>Wetter</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_11">
         <item>
          <layout class="QVBoxLayout" name="weatherVerticalLayout">
           <item alignment="Qt::AlignTop">
            <widget class="QGroupBox" name="weatherGeneralGroupBox">
             <property name="title">
              <string>Allgemein</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_15">
              <item>
               <layout class="QVBoxLayout" name="weatherGeneralVerticalLayout">
                <item>
                 <widget class="QCheckBox" name="enableWeatherCheckBox">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string>Wetterbericht aktivieren</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item alignment="Qt::AlignTop">
            <widget class="QGroupBox" name="weatherLocationGroupBox">
             <property name="title">
              <string>Ort</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_17">
              <item>
               <layout class="QVBoxLayout" name="weatherLocationVerticalLayout">
                <item>
                 <widget class="QComboBox" name="weatherLocationComboBox">
                  <property name="editable">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="searchLocationButton">
                  <property name="text">
                   <string>Suchen</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="selectLocationButton">
                  <property name="text">
                   <string>Auswählen</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="selectedLocationLabel">
                  <property name="text">
                   <string>Bitte wählen Sie einen Ort aus.</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="weatherVerticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
